hudi-cli/src/main/java/org/apache/hudi/cli/commands/RepairsCommand.java

```diff
@@ -123,7 +121,7 @@ public String addPartitionMeta(
         client.getActiveTimeline().getCommitTimeline().lastInstant().get().getTimestamp();
     List<String> partitionPaths =
         FSUtils.getAllPartitionFoldersThreeLevelsDown(HoodieCLI.storage, client.getBasePath());
-    StoragePath basePath = new StoragePath(client.getBasePath());
+    StoragePath basePath = client.getBasePathV2();
```

Comment:
Could you create a JIRA to remove `getBasePathV2()` and return `StoragePath` from `getBasePath()` as a follow-up?

Reply:
https://issues.apache.org/jira/browse/HUDI-7747